src/providers/test/json-rpc-provider/get-network.test.ts

```diff
@@ -1,18 +1,27 @@
 import { ethers } from 'ethers';
 import { JsonRpcProvider } from '../../../index';
 import { fakeUrls, rpcUrls } from '../rpc-urls';
+import { Networkish } from './../../../types/Network.types';
 
 const xdaiRPCUrl = rpcUrls.gno;
 const bscRPCUrl = rpcUrls.bnb;
 
 describe('provider.getNetwork happy path', () => {
-  async function testNetwork(rpcUrl: string) {
+  async function testNetwork(rpcUrl: string, network?: Networkish) {
     const essentialEth = new JsonRpcProvider(rpcUrl);
     const ethersProvider = new ethers.providers.StaticJsonRpcProvider(rpcUrl);
-    const [eeNetwork, ethersNetwork] = await Promise.all([
-      essentialEth.getNetwork(),
-      ethersProvider.getNetwork(),
-    ]);
+    let eeNetwork, ethersNetwork;
+    if (network) {
+      [eeNetwork, ethersNetwork] = await Promise.all([
+        essentialEth.getNetwork(network),
+        ethers.providers.getNetwork(network),
```

Comment:
@dawsbot you'll notice on here, comparing to the test below, that `ethers` actually has two separate `getNetwork` functions: one called on a provider to get the network that provider is using, and another more util-like function to get the information about a specified network. Given this, I'm thinking that it'd be better for us to split them up as well. If that plan makes sense, it seems like the easiest approach is to add this as a new util fn, instead of how `ethers` exports their `providers` package

Thoughts?